.jenkins

```diff
@@ -148,7 +148,7 @@ pipeline {
                                 -DCMAKE_BUILD_TYPE=Release \
                                 -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
                                 -DCMAKE_CXX_COMPILER=$WORKSPACE/bin/nvcc_wrapper \
-                                -DCMAKE_CXX_FLAGS=-Werror --Werror=all-warnings -Xcudafe --diag_suppress=3159 \
+                                -DCMAKE_CXX_FLAGS="-Werror --Werror=all-warnings -Xcudafe --diag_suppress=3159" \
```

Comment:
My gut feeling is that since this is one of the jobs that run on the first stage of the `Jenkins` CI, it's not a very good idea to disable device "warning as error" and defer potential issues to a later stage.